lib/Runtime/Library/JavascriptDate.cpp

```diff
@@ -1503,16 +1503,6 @@ namespace Js
             DateImplementation::DateTimeFlag::NoDate);
     }
 
-    // CONSIDER: ToGMTString and ToUTCString is the same, but currently the profiler use the entry point address to identify
-    // the entry point. So we will have to make the function different. Consider using FunctionInfo to identify the function
```

Comment:
Does F12 script profiler shows ToGMTString and ToUTCString after your change? if yes then its ok